test/core/integration/concrete/lockup/renounce/renounce.t.sol

```diff
@@ -76,16 +50,12 @@ abstract contract Renounce_Integration_Concrete_Test is Integration_Test, Lockup
         lockup.renounce(defaultStreamId);
     }
 
-    modifier whenCallerSender() {
-        _;
-    }
-
     function test_RevertGiven_NonCancelableStream()
         external
         whenNoDelegateCall
         givenNotNull
         givenWarmStream
-        whenCallerSender
+        whenCallerSender(users.sender)
```

Comment:
i personally like this suggestion https://github.com/sablier-labs/v2-core/pull/1058#discussion_r1819034860

Reply:
Curious why you prefer modifier names with argument over modifier names without argument? I think modifier names without argument are friendlier because the names are explicit enough.